packages/teleport/src/Console/useOnExitConfirmation.ts

```diff
@@ -66,11 +66,8 @@ function useOnExitConfirmation(ctx: ConsoleContext) {
   /**
    * confirmCloseSession prompts user to confirm to close.
    */
-  function confirmCloseSession(doc) {
-    if (!doc || !doc.sid) return false;
-
-    const numUsers = ctx.storeParties.state[doc.sid];
-    if (numUsers.length > 1) {
+  function confirmCloseSession(participants: number) {
```

Comment:
hm...there is no much value in this function now. It accepts an integer and checks for > 0.  You might very well delete it and move the alerting to the `verifyAndConfirm`